### PR TITLE
add generate doc

### PR DESCRIPTION
**Pull request by gene-zhan**
1. add typedoc
2. rename Receipt
3. remove crypto Instance

### CODE REVIEW

File: packages/libs/typedoc.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  // 输入选项 - 入口文件/文件夹
+  "entryPoints": [
+    "types","hostapi","components","common","proto"
+  ],
+  // 输入选项 - 排除文件/文件夹
```
Comment (dumbeng) [inline packages/libs/typedoc.json:6]: Do not include Chinese comments in JSON, this format is invalid.

File: packages/libs/components/context/tx.ts [hunk 2]
```diff
@@ -146,7 +146,7 @@ export class ReceiptContext {
 
   private constructor() {}
 
-  public get(): EthReceiptKey {
+  get content(): EthReceiptKey {
```
Comment (dumbeng) [inline packages/libs/components/context/tx.ts:149]: Field naming not clear. Align the names, why use `Tx` and `ReceiptContext` within the same scope? What's the difference between these two?

File: packages/libs/package.ts [hunk 3]
```diff
@@ -34,7 +34,6 @@ import {
 } from './common';
 
 export namespace sys {
-  export const crypto = CryptoApi.instance();
```
Comment (dumbeng) [inline packages/libs/package.ts:37]: Where is the crypto instance now?  Only saw it gets removed here. How do we access the crypto functions now?

Comment (gene-zhan) [reply packages/libs/package.ts:37]: Accessed via sys.hostApi.crypto
see https://docs.artela.network/develop/reference/aspect-lib/components/sys-hostapi#syshostapicrypto

File: packages/libs/components/context/tx.ts (hunk 2 above)
Comment (gene-zhan) [reply packages/libs/components/context/tx.ts:149]: sys.context.receipt(ctx).content.unwrap() is batter than sys.context.receipt(ctx).get().unwrap()

Comment (dumbeng) [reply packages/libs/components/context/tx.ts:149]: better to be sys.context.receipt(ctx).unwrap()

**Pull request merged**